### PR TITLE
Edge/cie 3492 documentation to handle UI development

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Review by bbyreddy — CHANGES_REQUESTED** [review]
Please check now if you can see comments.

File: content/edge/configuration-bundle/developing-an-application.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+weight: 85
+title: Developing applications
```
[the remark this answers is not in the document]
Comment (sharath-jk) [reply content/edge/configuration-bundle/developing-an-application.md:3]: Done

File: content/edge/configuration-bundle/developing-an-application.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+layout: redirect
+---
+
+If you are developing an application for the Edge appliance using a self-signed certificate, the self-signed certificate must be trusted by Node.js.
```
[the remark this answers is not in the document]
Comment (sharath-jk) [reply content/edge/configuration-bundle/developing-an-application.md:7]: Done

File: content/edge/configuration-bundle/developing-an-application.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+To add the Edge appliance's self-signed certificate to the Node.js trust store, set the value of the environment variable `NODE_EXTRA_CA_CERTS` to the path of the certificate on your host machine. For example:
```
[the remark this answers is not in the document]
Comment (sharath-jk) [reply content/edge/configuration-bundle/developing-an-application.md:9]: Done

**Review by bbyreddy — APPROVED** [review]

File: content/edge/configuration-bundle/developing-an-application.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+weight: 85
+title: Developing Cumulocity web applications
```
Comment (BeateRixen) [inline content/edge/configuration-bundle/developing-an-application.md:3]: Cumulocity > Cumulocity IoT, also below

File: content/edge/configuration-bundle/developing-an-application.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+set NODE_EXTRA_CA_CERTS=<path-to-the-certificagte>\certificate.pem
+```
+
+On Linux platforms
+
+```shell
+export NODE_EXTRA_CA_CERTS=<path-to-the-certificagte>/certificate.pem
+```
+
+ For more information about the Cumulocity web application development, see [web SDK guide](/web/overview/).
```
Comment (BeateRixen) [inline content/edge/configuration-bundle/developing-an-application.md:31]:
```suggestion
 For more information on  the {{< product-c8y-iot >}} web application development, see the [Web SDK guide](/web/overview/).
```

File: content/edge/configuration-bundle/developing-an-application.md (hunk 4 above)
Comment (BeateRixen) [inline content/edge/configuration-bundle/developing-an-application.md:3]:
```suggestion
title: Developing Cumulocity IoT web applications
```

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**